lasy/profiles/transverse/laguerre_gaussian_profile.py

```diff
@@ -6,36 +8,56 @@
 
 class LaguerreGaussianTransverseProfile(TransverseProfile):
     r"""
-    High-order Gaussian laser pulse expressed in the Laguerre-Gaussian formalism.
+    A high-order Gaussian laser pulse expressed in the Laguerre-Gaussian formalism.
+
+    Definition is according to Siegman "Lasers" pg. 646 eq. 64.
 
-    Class for an analytic profile.
-    More precisely, at focus (`z_foc=0`), the transverse envelope (to be used in the
+    More precisely, the transverse envelope (to be used in the
     :class:`.CombinedLongitudinalTransverseLaser` class) corresponds to:
 
     .. math::
+        \mathcal{T}(x, y) = \,
+                \mathcal{L}_{p,m} (x) \, \exp(i \Phi)
+
+    with
+
+    .. math::
+        \mathcal{L}_{p,m}(x) = A \left ( \frac{\sqrt{2}r}{w(z)} \right)^m l_{p,m} \left ( \frac{2 r^2}{w^2(z)} \right)
+        \exp{\left( -\frac{r^2}{w^2(z)}\right)} \exp{ \left ( -i k_0 \frac{r^2}{2 R(z)} \right )}
+
+        w(z) = w_{0} \sqrt{1 + \left( \frac{z}{Z_R}\right)^2}
 
-        \mathcal{T}(x, y) = r^{|m|}e^{-im\theta} \,
-        L_p^{|m|}\left( \frac{2 r^2 }{w_0^2}\right )\,
-        \exp\left( -\frac{r^2}{w_0^2} \right)
+        A = \frac{1}{w(z)} \sqrt{\frac{2 p!}{\pi (p + m)!}}
 
-    where :math:`x = r \cos{\theta}`,
-    :math:`y = r \sin{\theta}`, :math:`L_p^{|m|}` is the
-    Generalised Laguerre polynomial of radial order :math:`p` and
-    azimuthal order :math:`|m|`
+        R(z) = z + \frac{Z_R^2}{z}
+
+        \Phi(z) = \left(2 p + m + 1\right) \tan^{-1}\left({\frac{z}{Z_R}}\right)
```

Comment:
Just to avoid confusion between cotan and arctan
```suggestion
        \Phi(z) = \left(2 p + m + 1\right) \arctan\left({\frac{z}{Z_R}}\right)
```